lib/resources/collection/index.js

```diff
@@ -448,7 +448,7 @@ Collection.prototype.save = function (ctx, fn) {
 
   domain.changed =  function (property) {
```

Comment:
Shall we 
```js
                    var args = Array.prototype.slice.call(arguments);
                    for (var i = 0, ii = args.length;  i < ii; i++) {

// The original code...

                    }
                    return false;
```

So that we can call ```changed('fieldA', 'fieldB')``` to detect ```fieldA``` OR ```fieldB``` is changed

Why OR?
In one-to-many or many-to-many relationship, actions are more likely to be triggered by sub-set of fields on a item.